src/lumigo_tracer/utils.py

```diff
@@ -161,3 +164,71 @@ def is_aws_environment():
     :return: heuristically determine rather we're running on an aws environment.
     """
     return bool(os.environ.get("LAMBDA_RUNTIME_DIR"))
+
+
+def extract_frames_from_exception() -> List[dict]:
```

Comment:
I Think its we will be  better than this function will get the `inspect` via param instead of that scopes as magic